src/program/guide.njk

```diff
@@ -0,0 +1,194 @@
+  <p class="text-sm md:text-lg leading-relaxed mb-2 md:mb-4 text-gray-700"><strong class="text-gray-700 font-semibold">⚠️ Note:</strong> Seats are limited. Registration is compulsory.</p>
+  <p class="text-sm md:text-lg leading-relaxed mb-2 md:mb-4 text-gray-700"><strong class="text-gray-700 font-semibold">📅 When:</strong> <span class="text-gray-600 block sm:inline ml-1 text-base">15 September 2025, 9:00 AM-4:00 PM</span></p>
+  <div class="flex justify-center lg:justify-start pt-4">
+    {{ button(text="Register for Dev Sprint!", url="/program/dev-sprint", lime_bg=true, env = env) }}
+  </div>
+</div>
+
+<div class="bg-white p-4 md:p-6 my-4 md:my-6 rounded-lg border border-gray-200">
+  <h3 class="text-xl md:text-2xl font-semibold text-gray-700 mb-3 mt-6"><span class="text-gray-700 font-semibold">Open Spaces</span></h3>
+  <p class="text-sm md:text-lg leading-relaxed mb-2 md:mb-4 text-gray-700">Open Spaces at PyCon India brings together a community of Python enthusiasts, ranging from beginners to seasoned developers, educators, researchers, and organizers. It is a participant-driven meetup environment where anyone can lead or join spontaneous discussions. No formal presentations required.</p>
+  <p class="text-sm md:text-lg leading-relaxed mb-2 md:mb-4 text-gray-700"><strong class="text-gray-700 font-semibold">📅 When:</strong> <span class="text-gray-600 block sm:inline ml-1 text-base">13 & 14 September 2025</span></p>
+  <p class="text-sm md:text-lg leading-relaxed mb-2 md:mb-4 text-gray-700"><strong class="text-gray-700 font-semibold">📍 Where:</strong> <span class="text-gray-600 block sm:inline ml-1 text-base">1st Floor, NIMHANS Convention center</span></p>
+  <div class="flex justify-center lg:justify-start pt-4">
+    {{ button(text="Learn More", url="https://in.pycon.org/2025/open-spaces/", lime_bg=true, env = env) }}
+  </div>
+</div>
+
+<div class="bg-white p-4 md:p-6 my-4 md:my-6 rounded-lg border border-gray-200">
+  <h3 class="text-xl md:text-2xl font-semibold text-gray-700 mb-3 mt-6"><span class="text-gray-700 font-semibold">PyLadies Lunch</span></h3>
+  <p class="text-sm md:text-lg leading-relaxed mb-2 md:mb-4 text-gray-700">PyLadies is an international mentorship group focusing on helping more women and gender minorities become active participants and leaders in the Python open source community.</p>
+  <p class="text-sm md:text-lg leading-relaxed mb-2 md:mb-4 text-gray-700"><strong class="text-gray-700 font-semibold">📅 When:</strong> <span class="text-gray-600 block sm:inline ml-1 text-base">13 September 2025, 12:40-2:00 PM</span></p>
+  <p class="text-sm md:text-lg leading-relaxed mb-2 md:mb-4 text-gray-700"><strong class="text-gray-700 font-semibold">📍 Where:</strong> <span class="text-gray-600 block sm:inline ml-1 text-base">Open Space</span></p>
```

Comment:
Instead of Open Space, let's make it Board Room

Reply:
Done 👍🏾